pages/about/policies.html

```diff
@@ -299,7 +299,8 @@ <h1 class="ui header">
                   Secondary School Literacy Policy
                 </h1>
                 <p>
-                    Preview Content goes here
+                    <strong><em>Secondary School Literacy Policy</em></strong> "Mount Sion CBS Secondary School is an inclusive boys&rsquo; only school which values the abilities and achievements of all its pupils. Our school accepts the fundamental principle that literacy is the key to improving learning and raising standards. The teaching of literacy is not the sole responsibility of the English Department; it is the responsibility of every teacher within the school. Literacy is one of the cornerstones of education and it enables pupils to gain access to the school curriculum, to read for information and pleasure, and to communicate effectively...
```

Comment:
@denispop9 when I load the screen this text takes up more space than the other sections. Can you make it to match the others?

Reply:
ye no problem